src/docs/product/issues/performance-issues/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "Performance Issues"
+sidebar_order: 15
+description: "Learn more about Performance Issues and how to use them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+The **Issue Details** page for performance issues helps you to gain further insight into the source of an issue and the impact it has on your application's users. The main area of the page displays information about a specific transaction event that's part of an issue. The top panel and right-hand sidebar of the page display a summary of all the events grouped together in this issue:
+
+![Performance issue](performance-issue-details.png)
+
+<Note>
+
+You can find performance issues by searching `issue.category:performance` in **Issues**, **Discover**, and **Dashboards**.
+
+</Note>
+
+While the **Issue Details** page displays information about a specific transaction event that's part of an issue, you can navigate between the events of an issue using the "Older" and "Newer" buttons. You'll see information about the issue as well as the details of the most recent event. You'll be able to see the summary of when the issue was seen, a breakdown of the tags, span evidence, and breadcrumbs if they are available.
+
+## How Performance Issues Work
+
+Sentry detects performance issues by scanning incoming transaction events. First we check various properties of the transaction (span durations, span arrangement, span types, and so on) to detect likely problems. We then check these properties against corresponding thresholds to determine whether a performance issue exists. Lastly, we generate a unique problem fingerprint based on the problem type and location in the application, and use that fingerprint to create a performance issue.
+
+## Tags
+
+Tags are key/value string pairs that are both indexed and searchable. For example, a tag provides you with information such as the browser, device, or user associated with the event. Tags are the diagnostic information sent by the SDK for the individual event.
+
+The tags displayed in the main section of this page are specific to the event that you're viewing. In contrast, the tags displayed in the right-hand sidebar are a summary of all tag values for all events included in the issue. You can set your own tags to make them more useful for debugging as described in Customize Tags.
+
+## Span Evidence
+
+Span evidence is information that explains the performance problem in the context of the current event. It may include details like the specific database queries that cause the problem, the spans that contain the problem, and the slow spans that have an impact on performance. This section also has a span tree which shows how the offending spans relate to the rest of the event. Different performance problems will have slightly different kinds of evidence.
```

Comment:
This is better, much clearer. The word "offending" is maybe a little too fancy here. 

```suggestion
Span evidence is information that explains the performance problem in the context of the current event. It may include details like the specific database queries that cause the problem, the spans that contain the problem, and the slow spans that have an impact on performance. This section also has a span tree which shows how the spans with the longest durations relate to the rest of the event. Different performance problems will have slightly different kinds of evidence.
```

Reply:
How do you feel about "problematic" or some other synonym? "with the longest duration" is probably too specific. Sometimes they're fast, there's just a _lot_ of them, etc.

Reply:
Was trying to avoid the word "problematic" but we've used in the docs before, so go for it. I can't think of a better word.